client/lib/faux-plugins.js

```diff
@@ -232,13 +243,13 @@ module.exports = function(roomName) {
             time: msg.get('time'),
             messageId: msg.get('id'),
             youtubeId: match[1],
+            youtubeTime: parseYoutubeTime(match[3] || "0"),
             title: msg.get('content'),
           }
         })
         .filter(Boolean)
         .sortBy(video => video.time)
         .last()
-
```

Comment:
Oh, I added some logging, then removed, and i removed too many lines. Will re-add it.